packages/react-reconciler/src/__tests__/ReactIncrementalErrorHandling-test.internal.js

```diff
@@ -232,7 +232,12 @@ describe('ReactIncrementalErrorHandling', () => {
     expect(ReactNoop.getChildren()).toEqual([span('Caught an error: oops!')]);
   });
 
-  it("retries at a lower priority if there's additional pending work", () => {
+  it("retries at a lower priority if there's additional pending work", async () => {
+    if (gate(flags => flags.enableDiscreteEventMicroTasks)) {
```

Comment:
@acdlite there are 3 tests disabled in this file, which I'm not sure how to fix without an microtask integration with `toFlushAndYieldThrough`